core/ecschema-rpc/integration-tests/common/Settings.ts

```diff
@@ -0,0 +1,187 @@
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { LogLevel } from "@bentley/bentleyjs-core";
+import { IModelReadRpcInterface } from "@bentley/imodeljs-common";
+import { TestUserCredentials } from "@bentley/oidc-signin-tool";
+import { SchemaRpcInterface } from "@bentley/schema-rpcinterface-common";
+
+// tslint:disable:ter-indent
+
+export interface Backend {
+  version: string;
+  location: string;
+  name: string;
+  path: string;
+}
+
+export interface IModelData {
+  id: string;
+  projectId: string;
+  name?: string; // The name is not required to actually get the iModel, only the id.
+  changeSetId?: string;
+}
+
+export function getRpcInterfaces() {
+  const rpcInterfaces = [];
+  rpcInterfaces.push(IModelReadRpcInterface);
+  rpcInterfaces.push(SchemaRpcInterface);
+
+  return rpcInterfaces;
+}
+
+function checkEnabled(envVariable: string | undefined): boolean {
+  if (undefined === envVariable)
+    return false;
+
+  const regex = /true/i;
+  return regex.test(envVariable);
+}
+
+export class Settings {
+  private _backend: Backend = {} as Backend;
+  public env: number = 0;
+  public oidcClientId!: string;
+  public oidcScopes!: string;
+  public oidcRedirect!: string;
+  public imsUrl!: string;
+  public discovery!: string;
+  public gprid?: string;
+  public logLevel?: number;
+  public users: TestUserCredentials[] = [];
+
+  public iModels: IModelData[] = [];
+  public get iModel(): IModelData { return this.iModels[0]; }
+  public get writeIModel(): IModelData { return this.iModels[1]; }
+  public get user(): TestUserCredentials { return this.users[0]; }
+
+  public get Backend(): Backend { return this._backend; }
+
+  public get runiModelTileRpcTests(): boolean { return checkEnabled(process.env.RPC_IMODELTILE_ENABLE); }
+  public get runPresentationRpcTests(): boolean { return checkEnabled(process.env.RPC_PRESENTATION_ENABLE); }
+  public get runiModelReadRpcTests(): boolean { return checkEnabled(process.env.RPC_IMODELREAD_ENABLE); }
+  public get runiModelWriteRpcTests(): boolean { return checkEnabled(process.env.RPC_IMODELWRITE_ENABLE); }
+  public get runDevToolsRpcTests(): boolean { return checkEnabled(process.env.RPC_DEVTOOLS_ENABLE); }
+
+  constructor(env: NodeJS.ProcessEnv) {
+    const isFrontend = (typeof (process) === "undefined");
+    if (!isFrontend && undefined === env.TF_BUILD) {
+      const path = require("path");
+      const dotenv = require("dotenv");
+      const dotenvExpand = require("dotenv-expand");
+      // First check in process.cwd() for the config
+      let result = dotenv.config();
+      if (result.error) {
+        const potential = path.resolve(process.cwd(), "..", "..", "imodeljs-config", ".env");
```

Comment:
Can you remove the `imodeljs-config` support?  The `.env` file should always be expected as a peer of the `package.json` file if it's run locally.

Reply:
sure thing. So I changed the if body statement to just throw the result.error immediately, instead of checking if a imodeljs-config.env exists. Is that okay?